auslib/web/admin/swagger/api.yaml

```diff
@@ -1893,90 +1906,970 @@ paths:
+          description: request body for sc rule
+          required: true
+          schema:
+            type: object
+            allOf:
+              - $ref: '#/definitions/CSRFModel'
+              - $ref: '#/definitions/SCChangeType'
+              - $ref: '#/definitions/SCTime'
+              - $ref: '#/definitions/SCUptake'
+              - $ref: '#/definitions/RulesSCNullable'
+              - $ref: '#/definitions/RulesBase'
+      responses:
+        '200':
+          description: Returns sc_id of change
+          schema:
+            $ref: '#/definitions/SCIdModel'
+        '400':
+          $ref: '#/responses/invalidFormData'
 
-  CountModel:
-    title: Count
-    description: count of total objects/elements fetched
-    type: object
-    required:
-      - count
-    properties:
-      count:
-        description: count of total values fetched from DB
-        type: integer
-        minimum: 0
-        format: int32
-        example: 1
+  /scheduled_changes/permissions:
+    get:
+      summary: Returns scheduled changes for permissions.
+      description: >
+        Endpoint to get Scheduled Changes for Permissions. [Docs](http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#scheduled-changes)
+      tags:
+        - Scheduled Changes
+      operationId: auslib.web.admin.views.mapper.scheduled_changes_permissions_get
+      consumes: []
+      produces:
+        - application/json
+      externalDocs:
+        url: "http://mozilla-balrog.readthedocs.io/en/latest/admin_api.html#scheduled-changes-object"
+        description: Returns scheduled changes for permissions.
+      parameters:
+        - $ref: '#/parameters/allSCParam'
+      responses:
+        '200':
+          description: Returns JSON object of list of all scheduled changes for permissions plus total count
+          schema:
+            type: object
+            required:
+              - scheduled_changes
+            allOf:
+              - $ref: "#/definitions/CountModel"
+              - type: object
+                properties:
+                  scheduled_changes:
+                    description: array where each element is a scheduled change for a Permission.
+                    type: array
+                    minItems: 0
+                    items:
+                      type: object
+                      allOf:
+                        - $ref: '#/definitions/SCChangeType'
+                        - $ref: '#/definitions/SCTime'
+                        - $ref: '#/definitions/SCIdModel'
+                        - $ref: '#/definitions/SCDataVersion'
+                        - $ref: '#/definitions/SCGetModel'
+                        - $ref: '#/definitions/DataVersionNullable'
+                        - $ref: '#/definitions/OptionsModel'
+                        - $ref: '#/definitions/PermissionsModel'
+                      required:
+                        - change_type
+                        - data_version
+                        - when
```

Comment:
OK, this inconsistency is a bit confusing, but it's not a blocker. I think in the long term it might be a good idea to keep "required" out of the models, and make it explicit in every endpoint (even if that means repeating it in places).